src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
+import propTypes from 'prop-types';
+
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <>
+    <div className="movies">
+      {movies.map(movie => (
+        <div className="card" key={movie.imdbId}>
```

Comment:
I'm not sure, but I think, that `div class="card"` must be related with MovieCard.